### PR TITLE
Add GitHub Actions and fix some failing checks

### PR DESCRIPTION
**Pull request by sharlagelfand**
Just a draft for now to add GitHub Actions (a good time to learn, and super easy - of course - thanks to `usethis`! ), but there are some outstanding issues (you can see the checks are [failing](https://github.com/Sage-Bionetworks/magora/actions/runs/145329475)). 

All the errors/warnings/notes are due to the `phenotypes` data set not being checked into GitHub - e.g. the examples fail because they use `phenotypes` which isn't present and there's warnings/notes about it being documented but not existing. If it makes the most sense right now I can just remove the example/documentation until the data is finalized (and I'm not sure if the app will end up reading the data from Synapse each session vs prepping it in `data-raw/` anyways).

### CODE REVIEW

**Comment by sharlagelfand** [comment]
Ah, just saw the email with the new data! I'll update it then the issues I mentioned above shouldn't be an issue.

File: .github/workflows/R-CMD-check.yaml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        config:
+          - {os: macOS-latest,   r: 'devel',   vdiffr: false}
+          - {os: macOS-latest,   r: 'release', vdiffr: true}
+          - {os: windows-latest, r: 'release', vdiffr: true}
+          - {os: ubuntu-16.04,   r: 'release', vdiffr: true,  rspm: "https://packagemanager.rstudio.com/cran/__linux__/xenial/latest"}
+          - {os: ubuntu-16.04,   r: 'oldrel',  vdiffr: false, rspm: "https://packagemanager.rstudio.com/cran/__linux__/xenial/latest"}
+          - {os: ubuntu-16.04,   r: '3.5',     vdiffr: false, rspm: "https://packagemanager.rstudio.com/cran/__linux__/xenial/latest"}
+          - {os: ubuntu-16.04,   r: '3.4',     vdiffr: false, rspm: "https://packagemanager.rstudio.com/cran/__linux__/xenial/latest"}
+          - {os: ubuntu-16.04,   r: '3.3',     vdiffr: false, rspm: "https://packagemanager.rstudio.com/cran/__linux__/xenial/latest"}
+          - {os: ubuntu-16.054,  r: '3.2',     vdiffr: false, rspm: "https://packagemanager.rstudio.com/cran/__linux__/xenial/latest"}
```
Comment (karawoo) [inline .github/workflows/R-CMD-check.yaml:30]: Is this supposed to be 16.054, or 16.04 like the others? I noticed this build is failing with not much info, so I'm wondering if this is a typo and 16.054 is not an ubuntu version that github actions can find?

Comment (sharlagelfand) [reply .github/workflows/R-CMD-check.yaml]: Ah, I must have inputted an extra character when I copied it from your `sagethemes` check file, thank you! I was wondering why it was failing. I'll fix the commit I just made that removes it.

**Comment by sharlagelfand** [comment]
I think this PR is ready now! Closes #2, closes #6. The most recent check failed but only the misspecified ubuntu version (https://github.com/Sage-Bionetworks/magora/actions/runs/147835787) but I assume the fixed one will pass given that all the other checks did.

**Pull request merged**